pkg/services/org/orgimpl/stats.go

```diff
@@ -0,0 +1,46 @@
+package orgimpl
+
+import (
+	"context"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	ExporterName = "grafana"
+)
+
+var (
+	// MStatTotalUserAccountsNoRole is a metric gauge for total number of
+	// user accounts with no basic role
+	MStatTotalUserAccountsNoRole prometheus.Gauge
```

Comment:
prom metrics are expected to be computed more often than every 24h, (access control uses 10min).

You can check the exmaple of AccessControlStore on how it's handled

Reply:
Logic has been moved to User implementation